<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Shantha" ignored="2" name="Test" tests="2" failures="1" timestamp="18 Nov 2017 22:07:41 GMT" time="31.73" errors="0">
  <testcase name="testInvalidLogin" time="11.131" classname="com.actitime.tests.InvalidLoginTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at com.actitime.generic.Basepage.verifyTitle(Basepage.java:31)
	at com.actitime.tests.InvalidLoginTest.testInvalidLogin(InvalidLoginTest.java:18)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testInvalidLogin -->
  <testcase name="testvalidloginlogout" time="0.433" classname="com.actitime.tests.ValidLoginTest"/>
  <testcase name="@AfterTest postConditions" time="0.0" classname="com.actitime.generic.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterTest postConditions -->
  <testcase name="@BeforeTest preCondition" time="0.0" classname="com.actitime.generic.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeTest preCondition -->
</testsuite> <!-- Test -->
